ocs_ci/framework/conf/default_config.yaml

```diff
@@ -59,9 +59,8 @@ REPORTING:
   polarion:
     project_id: "OpenShiftContainerStorage"
   # Upstream: 'US' or Downstream: 'DS', used only for reporting (Test Run Name)
-  us_ds: 'US'
-  ocp_must_gather_image: "quay.io/openshift/origin-must-gather"
```

Comment:
why we are removing ocp image from default. we are still using ocp image for collecting logs from failed test cases

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/utils.py#L735

Reply:
Ah, good catch @vavuthu.  Removed by mistake when moving to upstream config.  Fixed in next commit